### PR TITLE
tuedontuoyo/get-16-initialize-the-react-application

### PR DESCRIPTION
**Pull request by 0tuedon**
Refixed the Eslint Configuration and Pretieer configurations and also added this on linear

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,70 @@
...
-The Front End Application for the get and share Recipe Application
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
```
Comment (Eronmmer) [inline README.md:9]: testing review with new rule added

**Review by Eronmmer — CHANGES_REQUESTED** [review]

**Comment by Eronmmer** [comment]
@asapconet @0tuedon more questions,
- What UI library/framework do you have in mind to use? 
- Don't you think it would be a good idea to configure/set up the dependencies for the above in this PR?

**Comment by asapconet** [comment]
> @asapconet @0tuedon more questions,
> 
> * What UI library/framework do you have in mind to use?
> * Don't you think it would be a good idea to configure/set up the dependencies for the above in this PR?

1.We are using only tailwind for no need for any complex tool than it.
2. Please accept this so first. we'll sort everything later. 

I just checked the PR now. I thought we already have a running app on the repo. 
Merge this first we'll decide on the rest of the things to be added thanks.

**Comment by Eronmmer** [comment]
Okay. sounds good @0tuedon can you resolve the 2 threads above so I can merge this PR?

**Comment by 0tuedon** [comment]
> Okay. sounds good @0tuedon can you resolve the 2 threads above so I can merge this PR?

Okay Chief

File: README.md (hunk 1 above)
Comment (0tuedon) [reply README.md:9]: I will Add that.

**Review by Eronmmer — APPROVED** [review]

**Pull request merged**